dataverse-webapp/src/main/webapp/editFilesFragment.xhtml

```diff
@@ -546,7 +546,7 @@
                                 </ui:fragment>
                             </div>
                         </p:column>
-                        <p:column class="col-file-action text-right">
+                        <p:column class="col-file-action text-right" style="width:130px !important;">
```

Comment:
I'm not an expert but shouldn't be size represented by %(or similar) based values?

Reply:
I've changed the width to be relative to the font size. Setting it to a certain % wouldn't work well on mobile, unless I specified a different % for different resolutions, but I fail to see the point.